docs/data/date-pickers/base-concepts/base-concepts.md

```diff
@@ -117,7 +117,9 @@ Make sure to provide an `aria-labelledby` prop to `popper` and/or `mobilePaper`
 
 ## TypeScript
 
-In order to benefit from the [CSS overrides](/material-ui/customization/theme-components/#theme-style-overrides) and [default prop customization](/material-ui/customization/theme-components/#theme-default-props) with the theme, TypeScript users need to import the following types.
+### Theme augmentation
+
+To benefit from the [CSS overrides](/material-ui/customization/theme-components/#theme-style-overrides) and [default prop customization](/material-ui/customization/theme-components/#theme-default-props) with the theme, TypeScript users need to import the following types.
```

Comment:
My IDE proposes to simplify `In order to" to "To", I can revert if you want but I agree that it's lighter to read

Reply:
So does the Grammarly app. 😉